components/HeaderBar.vue

```diff
@@ -49,6 +49,20 @@ export default Vue.extend({
   },
   methods: {
     rubles,
+    toggleSearcher () {
+      const searchModalWindow = document.getElementById('searcher');
+      const isSearcherOpen = (localStorage.getItem('isSearcherOpen') === 'true');
+
+      if (searchModalWindow) {
```

Comment:
Prefer to use guard clauses to decrease nesting
```suggestion
      if (!searchModalWindow) return;
```